### ISSUE TITLE
Facebook Ads failing to sync to Postgres destination

### ISSUE THREAD

**Issue by sooand** [airbytehq/airbyte#11984]
## Environment
- **Airbyte version**:  0.35.65-alpha
- **OS Version / Instance**: Ubuntu 20.04
- **Deployment**: Docker COMPOSE
- **Source Connector and version**: Facebook Marketing 0.2.41
- **Destination Connector and version**: Postgres 0.3.15
- **Severity**:  High 
- **Step where error happened**:  Sync job

## Current Behavior
Setting up Facebook Marketing to sync to Postgres results in following error when syncing: 
```
2022-04-13 18:19:38 ERROR i.a.w.DefaultReplicationWorker(run):169 - Sync worker failed.
java.util.concurrent.ExecutionException: io.airbyte.workers.DefaultReplicationWorker$DestinationException: Destination process exited with non-zero exit code 1
```
It has been tested with several different replication schemas, frequencies and sync modes.
Other sources to the same Destination works fine, example: LinkedIn Ads 0.1.6

## Expected Behavior
Sync should input the source data to the destination

## Logs

```

2022-04-13 16:08:32 [32mINFO[m i.a.w.w.WorkerRun(call):49 - Executing worker wrapper. Airbyte version: 0.35.65-alpha
2022-04-13 16:08:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):105 - Docker volume job log path: /tmp/workspace/15/1/logs.log
2022-04-13 16:08:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):110 - Executing worker wrapper. Airbyte version: 0.35.65-alpha
2022-04-13 16:08:32 [32mINFO[m i.a.w.DefaultReplicationWorker(run):104 - start sync worker. job id: 15 attempt id: 1
2022-04-13 16:08:32 [32mINFO[m i.a.w.DefaultReplicationWorker(run):116 - configured sync modes: {null.ads_insights_action_type=incremental - append_dedup, null.ads_insights=incremental - append_dedup}
2022-04-13 16:08:32 [32mINFO[m i.a.w.p.a.DefaultAirbyteDestination(start):69 - Running destination...
2022-04-13 16:08:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - Checking if airbyte/destination-postgres:0.3.15 exists...
2022-04-13 16:08:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - airbyte/destination-postgres:0.3.15 was found locally.
2022-04-13 16:08:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):106 - Creating docker job ID: 15
2022-04-13 16:08:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):158 - Preparing command: docker run --rm --init -i -w /data/15/1 --log-driver none --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e WORKER_CONNECTOR_IMAGE=airbyte/destination-postgres:0.3.15 -e WORKER_JOB_ATTEMPT=1 -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e AIRBYTE_VERSION=0.35.65-alpha -e WORKER_JOB_ID=15 airbyte/destination-postgres:0.3.15 write --config destination_config.json --catalog destination_catalog.json
2022-04-13 16:08:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - Checking if airbyte/source-facebook-marketing:0.2.41 exists...
2022-04-13 16:08:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - airbyte/source-facebook-marketing:0.2.41 was found locally.
2022-04-13 16:08:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):106 - Creating docker job ID: 15
2022-04-13 16:08:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):158 - Preparing command: docker run --rm --init -i -w /data/15/1 --log-driver none --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e WORKER_CONNECTOR_IMAGE=airbyte/source-facebook-marketing:0.2.41 -e WORKER_JOB_ATTEMPT=1 -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e AIRBYTE_VERSION=0.35.65-alpha -e WORKER_JOB_ID=15 airbyte/source-facebook-marketing:0.2.41 read --config source_config.json --catalog source_catalog.json
2022-04-13 16:08:32 [32mINFO[m i.a.w.DefaultReplicationWorker(lambda$getDestinationOutputRunnable$6):339 - Destination output thread started.
2022-04-13 16:08:32 [32mINFO[m i.a.w.DefaultReplicationWorker(run):158 - Waiting for source and destination threads to complete.
2022-04-13 16:08:32 [32mINFO[m i.a.w.DefaultReplicationWorker(lambda$getReplicationRunnable$5):279 - Replication thread started.
2022-04-13 16:08:33 [43mdestination[0m > SLF4J: Class path contains multiple SLF4J bindings.
2022-04-13 16:08:33 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2022-04-13 16:08:33 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2022-04-13 16:08:33 [43mdestination[0m > SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2022-04-13 16:08:33 [43mdestination[0m > SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2022-04-13 16:08:33 [44msource[0m > Starting syncing SourceFacebookMarketing
2022-04-13 16:08:33 [44msource[0m > Syncing stream: ads_insights 
2022-04-13 16:08:34 [44msource[0m > InsightAsyncJob(id=520559869685508, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-12 -> 2020-04-12]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:34 [43mdestination[0m > 2022-04-13 16:08:34 [32mINFO[m i.a.i.d.p.PostgresDestination(main):77 - starting destination: class io.airbyte.integrations.destination.postgres.PostgresDestination
2022-04-13 16:08:34 [43mdestination[0m > 2022-04-13 16:08:34 [32mINFO[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}
2022-04-13 16:08:34 [43mdestination[0m > 2022-04-13 16:08:34 [32mINFO[m i.a.i.b.IntegrationRunner(run):88 - Sentry transaction event: 10137ee3c5bc4f79912d2c52c21c369d
2022-04-13 16:08:34 [43mdestination[0m > 2022-04-13 16:08:34 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):106 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination
2022-04-13 16:08:34 [43mdestination[0m > 2022-04-13 16:08:34 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):107 - Command: WRITE
2022-04-13 16:08:34 [43mdestination[0m > 2022-04-13 16:08:34 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):108 - Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}
2022-04-13 16:08:34 [44msource[0m > InsightAsyncJob(id=504700921209624, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-13 -> 2020-04-13]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:35 [43mdestination[0m > 2022-04-13 16:08:35 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-04-13 16:08:35 [43mdestination[0m > 2022-04-13 16:08:35 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-04-13 16:08:35 [43mdestination[0m > 2022-04-13 16:08:35 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-04-13 16:08:35 [43mdestination[0m > 2022-04-13 16:08:35 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword multiline - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-04-13 16:08:35 [43mdestination[0m > 2022-04-13 16:08:35 [32mINFO[m i.a.i.b.s.SshTunnel(getInstance):170 - Starting connection with method: SSH_KEY_AUTH
2022-04-13 16:08:35 [44msource[0m > InsightAsyncJob(id=500825594845108, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-14 -> 2020-04-14]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:36 [44msource[0m > InsightAsyncJob(id=531780148291690, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-15 -> 2020-04-15]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [33mWARN[m o.a.s.c.k.StaticServerKeyVerifier(handleAcceptance):59 - Server at /46.101.113.129:22 presented unverified EC key: SHA256:h+K0D1zk9GbNmmQY/zqCK8KQHQn4R96pBtLbNj6S34k
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [32mINFO[m i.a.i.b.s.SshTunnel(openTunnel):287 - Established tunneling session.  Port forwarding started on /127.0.0.1:44855
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [32mINFO[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$toWriteConfig$0):96 - Write config: WriteConfig{streamName=facebook_ads_insights, namespace=null, outputSchemaName=public, tmpTableName=_airbyte_tmp_eld_facebook_ads_insights, outputTableName=_airbyte_raw_facebook_ads_insights, syncMode=append_dedup}
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [32mINFO[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$toWriteConfig$0):96 - Write config: WriteConfig{streamName=facebook_ads_insights_action_type, namespace=null, outputSchemaName=public, tmpTableName=_airbyte_tmp_kdn_facebook_ads_insights_action_type, outputTableName=_airbyte_raw_facebook_ads_insights_action_type, syncMode=append_dedup}
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [32mINFO[m i.a.i.d.b.BufferedStreamConsumer(startTracked):141 - class io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer started.
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [32mINFO[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):121 - Preparing tmp tables in destination started for 2 streams
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [32mINFO[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):125 - Preparing tmp table in destination started for stream facebook_ads_insights. schema: public, tmp table name: _airbyte_tmp_eld_facebook_ads_insights
2022-04-13 16:08:36 [44msource[0m > InsightAsyncJob(id=1904924166368463, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-16 -> 2020-04-16]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:36 [43mdestination[0m > 2022-04-13 16:08:36 [33mWARN[m o.a.s.c.u.l.LoggingUtils(warn):626 - globalRequest(ClientConnectionService[ClientSessionImpl[ploi@/46.101.113.129:22]])[hostkeys-00@openssh.com, want-reply=false] failed (SshException) to process: EdDSA provider not supported
2022-04-13 16:08:36 [44msource[0m > InsightAsyncJob(id=719526515885394, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-17 -> 2020-04-17]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:37 [43mdestination[0m > 2022-04-13 16:08:37 [32mINFO[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):125 - Preparing tmp table in destination started for stream facebook_ads_insights_action_type. schema: public, tmp table name: _airbyte_tmp_kdn_facebook_ads_insights_action_type
2022-04-13 16:08:37 [43mdestination[0m > 2022-04-13 16:08:37 [32mINFO[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):131 - Preparing tables in destination completed.
2022-04-13 16:08:37 [44msource[0m > InsightAsyncJob(id=574644943631529, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2020-04-18 -> 2020-04-18]>, breakdowns=[]): created AdReportRun
2022-04-13 16:08:38 [44msource[0m > InsightAsyncJob(id=402981107883788, <AdAccount> {
    "account_id": "1288290558215126",
    "id": "act_1288290558215126"
}, time_range=<Period [2022-04-13 -> 2022-04-13]>, breakdowns=[]): is 100 complete (Job Completed)
2022-04-13 18:19:35 [44msource[0m > Completed jobs: 5, Failed jobs: 0, Running jobs: 0
2022-04-13 18:19:37 [44msource[0m > Added: 0 jobs. Current throttle limit is MyFacebookAdsApi.Throttle(per_application=0.01, per_account=100.0), 0/100 job(s) in queue
2022-04-13 18:19:37 [44msource[0m > Read 1064 records from ads_insights_action_type stream
2022-04-13 18:19:37 [44msource[0m > Finished syncing ads_insights_action_type
2022-04-13 18:19:37 [44msource[0m > SourceFacebookMarketing runtimes:
Syncing stream ads_insights 1:05:40.516668
Syncing stream ads_insights_action_type 1:05:23.183577
2022-04-13 18:19:37 [44msource[0m > Finished syncing SourceFacebookMarketing
2022-04-13 18:19:37 [32mINFO[m i.a.w.DefaultReplicationWorker(lambda$getReplicationRunnable$5):305 - Total records read: 3592 (2 MB)
2022-04-13 18:19:37 [32mINFO[m i.a.w.DefaultReplicationWorker(run):163 - One of source or destination thread complete. Waiting on the other.
2022-04-13 18:19:37 [43mdestination[0m > 2022-04-13 18:19:37 [32mINFO[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):65 - Airbyte message consumer: succeeded.
2022-04-13 18:19:37 [43mdestination[0m > 2022-04-13 18:19:37 [32mINFO[m i.a.i.d.b.BufferedStreamConsumer(close):217 - executing on success close procedure.
2022-04-13 18:19:37 [43mdestination[0m > 2022-04-13 18:19:37 [32mINFO[m i.a.i.d.b.BufferedStreamConsumer(flushQueueToDestination):181 - Flushing buffer: 8145212 bytes
2022-04-13 18:19:37 [43mdestination[0m > 2022-04-13 18:19:37 [32mINFO[m i.a.i.d.b.BufferedStreamConsumer(lambda$flushQueueToDestination$1):185 - Flushing facebook_ads_insights: 1064 records
2022-04-13 18:19:38 [1;31mERROR[m i.a.w.DefaultReplicationWorker(run):169 - Sync worker failed.
java.util.concurrent.ExecutionException: io.airbyte.workers.DefaultReplicationWorker$DestinationException: Destination process exited with non-zero exit code 1
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at io.airbyte.workers.DefaultReplicationWorker.run(DefaultReplicationWorker.java:164) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
	at io.airbyte.workers.DefaultReplicationWorker.run(DefaultReplicationWorker.java:57) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$2(TemporalAttemptExecution.java:155) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
	Suppressed: io.airbyte.workers.WorkerException: Destination process exit with code 1. This warning is normal if the job was cancelled.
		at io.airbyte.workers.protocols.airbyte.DefaultAirbyteDestination.close(DefaultAirbyteDestination.java:119) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
		at io.airbyte.workers.DefaultReplicationWorker.run(DefaultReplicationWorker.java:126) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
		at io.airbyte.workers.DefaultReplicationWorker.run(DefaultReplicationWorker.java:57) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$2(TemporalAttemptExecution.java:155) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
		at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: io.airbyte.workers.DefaultReplicationWorker$DestinationException: Destination process exited with non-zero exit code 1
	at io.airbyte.workers.DefaultReplicationWorker.lambda$getDestinationOutputRunnable$6(DefaultReplicationWorker.java:354) ~[io.airbyte-airbyte-workers-0.35.65-alpha.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	... 1 more
2022-04-13 18:19:38 [32mINFO[m i.a.w.DefaultReplicationWorker(run):228 - sync summary: io.airbyte.config.ReplicationAttemptSummary@3f3b157e[status=failed,recordsSynced=2128,bytesSynced=2462975,startTime=1649866112524,endTime=1649873978202,totalStats=io.airbyte.config.SyncStats@123c4717[recordsEmitted=2128,bytesEmitted=2462975,stateMessagesEmitted=1464,recordsCommitted=0],streamStats=[io.airbyte.config.StreamSyncStats@857b880[streamName=facebook_ads_insights,stats=io.airbyte.config.SyncStats@3ef9c9a8[recordsEmitted=1064,bytesEmitted=1340903,stateMessagesEmitted=<null>,recordsCommitted=<null>]], io.airbyte.config.StreamSyncStats@598ebbec[streamName=facebook_ads_insights_action_type,stats=io.airbyte.config.SyncStats@297a5b35[recordsEmitted=1064,bytesEmitted=1122072,stateMessagesEmitted=<null>,recordsCommitted=<null>]]]]
2022-04-13 18:19:38 [32mINFO[m i.a.w.DefaultReplicationWorker(run):248 - Source output at least one state message
2022-04-13 18:19:38 [33mWARN[m i.a.w.DefaultReplicationWorker(run):261 - State capture: No state retained.
2022-04-13 18:19:38 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):131 - Stopping cancellation check scheduling...
2022-04-13 18:19:38 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$1):147 - sync summary: io.airbyte.config.StandardSyncOutput@2264ae45[standardSyncSummary=io.airbyte.config.StandardSyncSummary@38ff808[status=failed,recordsSynced=2128,bytesSynced=2462975,startTime=1649866112524,endTime=1649873978202,totalStats=io.airbyte.config.SyncStats@123c4717[recordsEmitted=2128,bytesEmitted=2462975,stateMessagesEmitted=1464,recordsCommitted=0],streamStats=[io.airbyte.config.StreamSyncStats@857b880[streamName=facebook_ads_insights,stats=io.airbyte.config.SyncStats@3ef9c9a8[recordsEmitted=1064,bytesEmitted=1340903,stateMessagesEmitted=<null>,recordsCommitted=<null>]], io.airbyte.config.StreamSyncStats@598ebbec[streamName=facebook_ads_insights_action_type,stats=io.airbyte.config.SyncStats@297a5b35[recordsEmitted=1064,bytesEmitted=1122072,stateMessagesEmitted=<null>,recordsCommitted=<null>]]]],state=<null>,outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@5a7642db[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@5258b57d[stream=io.airbyte.protocol.models.AirbyteStream@55c92ff5[name=facebook_ads_insights,jsonSchema={...},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=true,defaultCursorField=[date_start],sourceDefinedPrimaryKey=[[date_start], [account_id], [ad_id]],namespace=<null>,additionalProperties={}],syncMode=incremental,cursorField=[date_start],destinationSyncMode=append_dedup,primaryKey=[[date_start], [account_id], [ad_id]],additionalProperties={}]],additionalProperties={}],failures=[io.airbyte.config.FailureReason@56168b8c[failureOrigin=destination,failureType=<null>,internalMessage=io.airbyte.workers.DefaultReplicationWorker$DestinationException: Destination process exited with non-zero exit code 1,externalMessage=Something went wrong within the destination connector,metadata=io.airbyte.config.Metadata@4a3ca03c[additionalProperties={attemptNumber=1, jobId=15}],stacktrace=java.util.concurrent.CompletionException: io.airbyte.workers.DefaultReplicationWorker$DestinationException: Destination process exited with non-zero exit code 1
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: io.airbyte.workers.DefaultReplicationWorker$DestinationException: Destination process exited with non-zero exit code 1
	at io.airbyte.workers.DefaultReplicationWorker.lambda$getDestinationOutputRunnable$6(DefaultReplicationWorker.java:354)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	... 3 more
,retryable=<null>,timestamp=1649873978200]]]
2022-04-13 18:19:38 [32mINFO[m i.a.w.t.TemporalUtils(withBackgroundHeartbeat):235 - Stopping temporal heartbeating...
2022-04-13 18:19:38 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):56 - JSON schema validation failed. 
errors: $.tunnel_method: must be a constant value NO_TUNNEL

```

</details>

## Steps to Reproduce
1. setup Facebook Marketing Source
2. setup Postgres Destination
3. Start connection between these

**Comment by marcosmarxm** [comment]
Hello @sooand I'm closing due inactivity. Please test latest version of Airbyte platform and connectors. If the issue persist please open a new issue.
